src/main/java/model/lotto/LottoStorage.java

```diff
@@ -0,0 +1,35 @@
+package model.lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import model.winningnumber.LottoWinningNumberDTO;
+import model.bonusball.BonusBallDTO;
+
+public class LottoStorage {
+    private List<Lotto> lottoStorage;
+
+    public LottoStorage(LottoCount lottoCount) {
+        this.lottoStorage = store(lottoCount);
+    }
+
+    private List<Lotto> store(LottoCount lottoCount) {
+        List<Lotto> lottos = new ArrayList<>();
+
+        while (!lottoCount.isZero()) {
+            lottos.add(new Lotto(RandomLottoNumbersGenerator.pickSixNumbers()));
+            lottoCount.makeLotto();
+        }
+        return lottos;
+    }
+
+    public List<LottoDTO> getLottoStorageDTO() {
+        List<LottoDTO> lottoDTOS = new ArrayList<>();
+        lottoStorage.forEach(lotto -> lottoDTOS.add(lotto.getLottoDTO()));
+        return lottoDTOS;
+    }
+
+    public void compare(BonusBallDTO bonusBallDTO, LottoWinningNumberDTO winningNumbersDTO) {
+        lottoStorage.forEach(lotto -> lotto.compare(bonusBallDTO, winningNumbersDTO));
+    }
+}
```

Comment:
다른곳은 newLine을 작성해두셨기 때문에 여기에도 지키면 좋겠네요 :) 
```suggestion
}

```

Reply:
해당 부분은 그동안 의식하지 못했던 부분이었네요..! 혹시 파일 마지막에 newLine을 추가하는 것은, 관례적인 걸까요?

Reply:
관례라긴보단, 통일성을 중시하시면 좋을거 같아요.
공개되어있는 Convention들을 찾아보시고, 그린론만의 생각을 한번 정리해보면 어떨까 합니다 :)

Reply:
네, 감사합니다!